website/docs/reference/sinks/sematext.md.erb

```diff
@@ -0,0 +1,21 @@
+<%- component = metadata.sinks.sematext -%>
+
+<%= component_header(component) %>
+
+## Configuration
+
+<%= component_config_example(component) %>
+
+## Options
```

Comment:
There should be a `## Requirements` section here. Rebase with `master` and look at the other component templates for an example.

Reply:
I am inline with master and when I regenerate this file it doesn't add this but I see it in the es version? Does this mean I need to add this manually?